src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -739,15 +739,6 @@
   "com.minecolonies.coremod.mercenary.mercenarystealcitizen": "%s is unhappy, a Mercenary took their %s!",
   "com.minecolonies.coremod.mercenary.stealbuilding": "A Mercenary just stole %s from a building!",
 
-  "com.minecolonies.coremod.gui.happiness.happinessmodifier": "Happiness Modifiers",
```

Comment:
"com.minecolonies.coremod.gui.happiness.happinessmodifier" is used in WindowCitizen.java.  The rest are unused.